openmls/src/treesync/node/leaf_node.rs

```diff
@@ -598,6 +598,20 @@ impl LeafNode {
             .credentials
             .contains(credential_type)
     }
+
+    /// Check whether the this leaf node supports all the required extensions
```

Comment:
I know it's only a move, but we could already fix the typo.

```suggestion
    /// Check whether this leaf node supports all the required extensions
```